tests/yelp_bytes_test.py

```diff
@@ -227,6 +227,20 @@ def test_to_bytes_is_like_str_encode(value):
     try:
         to_bytes_result = to_bytes(value, 'US-ASCII')
     except:
-        to_bytes_result = '(error)'
+        to_bytes_result = '(error)'  # pylint: disable=R0204
 
     assert bytes_result == to_bytes_result
+
+
+@pytest.mark.parametrize('value', (
+    UNICODE.ascii,
+    UNICODE.latin1,
+    UNICODE.win1252,
+    UNICODE.bmp,
+    UNICODE.utf8,
+    b"this is a bytestring",
+    u"this is a unicode string",
+    "native string"
```

Comment:
this file imports `unicode_literals` -- to be a native string you need `str("native string")`